ticket_upgrade_prediction/models/lasso/lasso.py

```diff
@@ -0,0 +1,79 @@
+import gc
+
+import mlflow
+import numpy as np
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import train_test_split
+
+from ticket_upgrade_prediction.models.base import BaseModel
+from ticket_upgrade_prediction.pipeline import Pipeline
+
+
+class LassoModel(BaseModel):
+    def __init__(self, from_mlflow: bool = False, **kwargs):
```

Comment:
Maybe lets iniciate it with `self.model = None`, and then depending on situation, use `.get_fitted_model(run_id=<run_id>)` /`.get_fitted_model(model_version=<version>, model_name=<model_name>),`  if we want to download model from mlflow, and `fit_model(self, **kwargs)` if we want to fit model.

In such a way, we can pass diffrent sets of arguments in each method, and it becomes much more readable, then using one `kwargs` dictionary from `__init__`  to supply arguments for two separate methods.

Reply:
not sure if i got it right, please check my last commit

Reply:
I showed what i mean here: 
https://github.com/kuba1302/ticket-upgrade-prediction/pull/11